ftl/core/deck-config.ftl

```diff
@@ -39,6 +39,11 @@ deck-config-tab-description =
     - `Preset`: The limit is shared with all decks using this preset.
     - `This deck`: The limit is specific to this deck.
     - `Today only`: Make a temporary change to this deck's limit.
+deck-config-new-cards-ignore-review-limit = New cards ignore review limit
+deck-config-new-cards-ignore-review-limit-tooltip =
+    By default, the review limit also applies to new cards and no new cards will be
+    shown when the review limit has been reached. If this option is enabled, new cards
+    will be shown regardless of the review limit.
```

Comment:
The documentation is why I've marked it as a draft.
How do we proceed with the `deck-config-limit-new-bound-by-reviews` message? Should we remove it because it is now covered in its own section and does not apply universally anymore? Not sure if the `deck-config-limit-interday-bound-by-reviews` message would look a bit out of place then.

Reply:
new-bound-by no longer seems necessary. interday-bound-by still has some value, though we should probably only show it when showing the review limit, and perhaps remove the ", and finally new cards". WDYT?

Reply:
Sounds good. Should I add a new message to drop ", and finally new cards" or does changing the English one suffice?